src/renderer/components/utils/WebFrame.js

```diff
@@ -43,20 +43,30 @@ export default {
       type: Number,
       required: false,
       default: 150
+    },
+    classes: {
+      type: String,
+      required: false,
+      default: ''
     }
   },
 
   render (h, ctx) {
     const src = ctx.props.src
     const url = isValidURL(src) ? src : 'about:blank'
 
-    return h('div', [
+    return h('div', {
+      attrs: {
```

Comment:
I think we can remove this div, iframe won't inherit custom attributes because this component is functional (https://github.com/vuejs/vue/issues/8604#issuecomment-410441704).

Reply:
🤔 	good point. it would make it much easier